src/components/global/Column.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { Droppable } from "react-beautiful-dnd";
+import BoardTask from "./Task/BoardTask";
+
+function Column(props) {
+  const { column, tasks, subTasks, handleList } = props;
+
+  const [open, setOpened] = React.useState(false);
```

Comment:
`const [opened, setOpened]`
or
`const [isOpened, setIsOpened]`

are the conventions for boolean and useState